lib/properties.js

```diff
@@ -29,9 +29,8 @@ class Properties extends EventEmitter {
       // Get Dynamic layers' source instances
       .concat(this.layers.map((layer) => layer.source).filter((source) => !!source))
 
-      // Add the active View's sources. Reversed for precedence consistency with Layers:
-      // Entry 0 is most precedent, LAST is least precedent.
-      .concat(this.active.sources.reverse());
+      // Add the active View's sources.
+      .concat(this.active.sources);
```

Comment:
If we want to keep it unchanged we'd do:
```javascript
return []
  .concat(this.layers.map((layer) => layer.source).filter((source) => !!source))
  .concat(this.active.sources.slice().reverse());
```

Keep in mind the reversal was applying to the order in which active sources were listed in the `/v1/status` endpoint so we have to reverse `this.active.sources` before we `concat` it with the rest of the sources.

I'm ambivalent. Let me know your preference and we can do it however you want.